### ISSUE TITLE
Crash for using keyboard to operate this software

### ISSUE THREAD

**Issue by wahom** [files-community/Files#1532]
**Describe the bug**
When i using keyboard to forward, backward or modify file name, crash will happen every time.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS Version: [e.g. Windows 10 1903 18362.175]
 - App version: [v0.11.0]

**Additional context**
Add any other context about the problem here. Does this problem occur again after restarting the app?

**Comment by Jaiganeshkumaran** [comment]
@wahom Could you upload the log file? To access the log file, go to Files UWP Settings->About->Open log location.

**Comment by smallg0at** [comment]
Just reproduced by using F2 to rename a file (0.12.3.0):
```
2020-07-26 15:56:22.0098|INFO|Files.App|App launched|
2020-07-26 15:56:22.5912|INFO|Files.Filesystem.DrivesManager|Drive added: C:\, Fixed|
2020-07-26 15:56:30.7858|ERROR|Files.App|Delegate to an instance method cannot have null 'this'.|System.ArgumentException: Delegate to an instance method cannot have null 'this'.
   at SharedLibrary!<BaseAddress>+0x7e3276
   at Files.GenericFileBrowser.AllView_CellEditEnded(Object, DataGridCellEditEndedEventArgs) + 0x30
   at Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.OnCellEditEnded(DataGridCellEditEndedEventArgs) + 0x42
   at Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.EndCellEdit(DataGridEditAction, Boolean, Boolean, Boolean) + 0x427
   at Files.GenericFileBrowser.AllView_PreparingCellForEdit(Object, DataGridPreparingCellForEditEventArgs) + 0xdb
   at Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.OnPreparingCellForEdit(DataGridPreparingCellForEditEventArgs) + 0x42
   at Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.PreparingCellForEditPrivate(FrameworkElement) + 0xf2
   at Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.EditingElement_Loaded(Object, RoutedEventArgs) + 0xc3
   at SQLitePCL.delegate_function_aggregate_final.Invoke(sqlite3_context, Object) + 0x2e
   at __Interop.Intrinsics.HasThisCall__23[TArg0](Object, IntPtr, Object, TArg0) + 0x36
   at Files!<BaseAddress>+0x1298a60
   at System.Runtime.InteropServices.McgMarshal.ThrowOnExternalCallFailed(Int32, RuntimeTypeHandle) + 0x21
   at __Interop.ComCallHelpers.Call(__ComObject, RuntimeTypeHandle, Int32) + 0xb8
   at __Interop.ForwardComStubs.Stub_11[TThis](__ComObject, Int32) + 0x24
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object, UnhandledErrorDetectedEventArgs) + 0x3d
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object, UnhandledErrorDetectedEventArgs) + 0x75
   at System.EventHandler`1.Invoke(Object, TEventArgs) + 0x2e
   at __Interop.Intrinsics.HasThisCall__23[TArg0](Object, IntPtr, Object, TArg0) + 0x36
   at Files!<BaseAddress>+0x1298a60

2020-07-26 15:56:34.9962|INFO|Files.App|App launched|
2020-07-26 15:56:35.5533|INFO|Files.Filesystem.DrivesManager|Drive added: C:\, Fixed|
```

**Comment by yaichenbaum** [comment]
@smallg0at Can you try updating the app to v0.15.4 to check if the issue still exists?

**Comment by gave92** [comment]
Fixed in #1736
